runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomConfig.java

```diff
@@ -42,10 +43,16 @@ default boolean innerLimit()
 		return false;
 	}
 
+	int OUTER_LIMIT_MIN = -400;
```

Comment:
We generally put (implicit)  static/final fields at the top of classes/interfaces